frontend/packages/dev-console/integration-tests/support/constants/global.ts

```diff
@@ -25,3 +25,11 @@ export enum operators {
   knativeCamelOperator = 'knative Apache Camel Operator',
   eclipseCheOperator = 'Eclipse Che',
 }
+
+export enum resources {
+  deploymentconfigs = 'Deployment Configs',
+  buildconfigs = 'Build Configs',
+  services = 'Services',
+  imagestreams = 'Image Streams',
+  routes = 'Routes',
+}
```

Comment:
enum variables should start with Capital letters, please update this and appropriate files as well

Reply:
sure, done. thanks